### DISCUSSION TITLE
[Feature] special storage template variable {{filetype}} for live photo video and new variable {{sss}}

### DISCUSSION THREAD

**Discussion by sevtdy** [immich-app/immich#6545] [Feature Request]
### The feature

Current when I upload an live photo with template `{{y}}/{{MM}}/{{filetype}}_{{y}}{{MM}}{{dd}}_{{hh}}{{mm}}{{ss}}`

I will got 2 files:
`2022/02/IMG_20220203_045605.heic` and `2022/02/VID_20220203_045605.mov`

but for me, I would expect 2 files with the same name start with IMG and  :
`2022/02/IMG_20220203_045605.heic` and `2022/02/IMG_20220203_045605.mov`
this naming rule make more scene to me and many other backup software use it too, like NextCloud

Also I think it would be nice if we can have {{sss}} variable, this is useful for continuous shooting photos

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by 528-491** [comment]
> {{sss}} variable

For milliseconds?
+1

**Comment by sevtdy** [comment]
https://github.com/immich-app/immich/releases/tag/v1.99.0
